openid-sharedsignals-framework-1_0.md

```diff
@@ -887,7 +887,8 @@ Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
     "urn:example:secevent:events:type_2",
     "urn:example:secevent:events:type_3",
     "urn:example:secevent:events:type_4"
-  ]
+  ],
+  "format": "email"
```

Comment:
1. `format` describes how a subject claim should be interpreted. This is already part of the event. This format is abstracted away from the stream configuration. 
2. Stream configuration should define what events it can produce. Adding `format` limits the usage of events in the stream. 

eg. Existing events define `subject` as complex objects like 

```
"subject": {
                "device": {
                    "format": "iss_sub",
                    "iss": "https://idp.example.com/123456789/",
                    "sub": "e9297990-14d2-42ec-a4a9-4036db86509a"
                },
                "tenant": {
                    "format": "opaque",
                    "id": "123456789"
                }
```

In this example an event has 2 subjects of different formats. Adding `format` in the stream will completely break existing event processing as it is more restrictive and not cleanly abstracted.

I do NOT think adding `format` in stream configuration is a clean abstraction

Reply:
I'm trying to remember when this change was discussed in the WG meetings. Do you have a pointer to the notes of that meeting?

Reply:
Thanks for clarifying things for me @FragLegs! 

Just like "subjects" are getting removed from the status, I think it would add the value to fix this abstraction/layering issue before the spec is sent for approval. On that note, it may not be needed to update examples at this point.